midi_util.py

```diff
@@ -105,35 +106,44 @@ def midi_decode(pattern,
     # Extract all tracks at highest resolution
     merged_notes = None
```

Comment:
I think we don't need to compute merged notes anymore. Merged notes is simply np.ceil(merged_volume). This speeds up caching process...